docs/projects/P2/3_secondsprint.md

```diff
@@ -0,0 +1,108 @@
+
+Being able to communicate effectively is an important software engineering skill. As such, your reflection should be well-written and easy to read. Be sure to leave time after writing for revision and proofreading.
+
+There are many convenient tools for collaborative text editing; for instance, the course staff used Google Docs for this homework document. We recommend using these tools to draft your answers; then, submit your final answers to Gradescope.
+
+
+## Individual Reflection (30 pts)
+
+In addition to the team reflection, you will also be submitting an individual reflection on Gradescope. Specifically, we will ask you to reflect on the process, scheduling, and teamwork, and connect this project experience with your previous experience with collaborative development. Your previous experience may be from an academic or non-academic setting, such as internships, hackathons, or personal projects.
+
+Your reflection should answer the following questions: 
+
+- What previous collaborative projects have you done before? How does this project experience align with your previous experiences? What was similar? What was different?
+- What did you personally learn from this project’s development process? Was there anything unexpected?
+- What are you planning to do differently or improve upon in your future projects? What would you like to keep doing?
+
+Similar to the team reflection task, we will grade the quality and depth of your reflection.
+
+## Grading
+To receive full credit for the implementation, we expect:
+
+- [ ] **Progress from Interim Snapshot**
+    - [ ] Fulfillment of implementation goals as outlined by the planned milestones, **or** a clearly written justification in the milestone description of why elements fell through
+    - [ ] An updated Project Board for the final progress
+    - [ ] Reasonable code structure and style, including documentation where appropriate
+    - [ ] Coherent commits of reasonable size with meaningful commit messages by all team members
+    - [ ] High quality usage of Git/GitHub tools, such as issues, milestones, pull requests, and commits
+- [ ] **Additional Requirements for Final Submission**
+    - [ ] Automated test cases for the implemented functionalities
+    - [ ] GitHub Actions running on the repository and passing on the main branch
+    - [ ] Created `UserGuide.md` file containing user documentation of the new feature that fulfills the guidelines given above
```

Comment:
- [ ] A functional new feature that works when following the steps in `UserGuide.md`. Note that we aren't expecting the feature to be bug free, but it should work with general use.